QModManager/API/ModLoading/Internal/QModPlaceholder.cs

```diff
@@ -0,0 +1,41 @@
+﻿namespace QModManager.API.ModLoading.Internal
+{
+    using System.Reflection;
+
+    internal class QModPlaceholder : QMod, IQMod
+    {
+        private readonly ModStatus status;
+        /// <summary>
+        /// The dummy <see cref="QMod"/> which is used to represent QModManager
+        /// </summary>
+        internal static QModPlaceholder QModManager { get; } = new QModPlaceholder
+        {
+            Id = "QModManager",
+            DisplayName = "QModManager",
+            Author = "The QModManager Dev Team",
```

Comment:
`the QModManager dev team` or `The QModManager Dev Team`?

Reply:
I'm partial to **T**itle **C**asing

Reply:
Then also change this in `AssemblyInfo.cs` (for all of the projects) and in `QModsInstallerScript.iss` please

Reply:
Or just change this to `QModManager`. Easier that way